src/openforms/js/components/admin/form_design/FormLogic.js

```diff
@@ -64,8 +65,18 @@ const FormLogic = ({logicRules = [], onChange, onDelete, onAdd}) => {
 
   const backendDataToLoad = [
     {endpoint: SERVICES_ENDPOINT, stateVar: 'services'},
-    // TODO load ServiceFetchConfigurations
-    {endpoint: '/api/v2/service-fetch-configurations', stateVar: 'serviceFetchConfigurations'},
+    {
+      endpoint: SERVICE_FETCH_CONFIG_ENDPOINT,
+      stateVar: 'serviceFetchConfigurations',
+      // TODO endpoint itself should probably do this?
```

Comment:
...  these `backendDataToLoad` entries are conceptually our own `serviceFetchConfiguration`s. We could apply the same naming and patterns here. Maybe not the same class names (that could be confusing), but the functions and attributes.

Reply:
I think it might actually be alright to keep this reshaping here, because the only reason this has to happen is to handle the data in the underlying React components. It would be strange to have the API return headers/queryParams as lists of tuples, while in the database it is still stored as json blobs, imo.